Samples/StressTests/Armonik.Samples.StressTests.Client/Program.cs

```diff
@@ -71,69 +72,70 @@ private static async Task Main(string[] args)
+                                           () => 50,
+                                           "specify the number of task per buffer"),
+                           new Option<int>("--nbBufferPerChannel",
+                                           () => 5,
+                                           "specify the number of concurrent buffer per channel"),
+                           new Option<int>("--nbChannel",
+                                           () => 5,
+                                           "specify the number of Grpc Channel"),
+                           new Option<string>("--jsonPath",
+                                              () => "",
+                                              "specify the jsonPath to get metrics. Options are Raw or Json"),
+                         };
+
+      pTaskCommand.Handler = CommandHandler.Create((ContainerOptions options) =>
+                                                   {
+                                                     logger_.LogInformation("Option Parallel task Run");
+                                                     logger_.LogInformation($"--nbTask              = {options.NbTask}");
+                                                     logger_.LogInformation($"--nbInputBytes        = {options.NbInputBytes}");
+                                                     logger_.LogInformation($"--nbOutputBytes       = {options.NbOutputBytes}");
+                                                     logger_.LogInformation($"--workLoadTimeInMs    = {options.WorkLoadTimeInMs}");
+                                                     logger_.LogInformation($"--partition           = {options.Partition}");
+                                                     logger_.LogInformation($"--nbTaskPerBuffer     = {options.NbTaskPerBuffer}");
+                                                     logger_.LogInformation($"--nbBufferPerChannel  = {options.NbBufferPerChannel}");
+                                                     logger_.LogInformation($"--nbChannel           = {options.NbChannel}");
+                                                     logger_.LogInformation($"--jsonPath              = {options.JsonPath}");
+
+
+                                                     var test1 = new StressTests(configuration_,
+                                                                                 factory,
+                                                                                 options.Partition,
+                                                                                 options.NbTaskPerBuffer,
+                                                                                 options.NbBufferPerChannel,
+                                                                                 options
+                                                                                   .NbChannel); //Bug:  SystemCommand line version cannot accept more than 8 arguments !!!
```

Comment:
https://learn.microsoft.com/en-us/dotnet/standard/commandline/model-binding#parameter-binding-more-than-8-options-and-arguments

Reply:
J'ai trouvé la solution mais j'ai oublié d'enlever le commentaire oups